pyworker/job.py

```diff
@@ -146,7 +148,7 @@ def set_error_unlock(self, error):
             # set new exponential run_at
             setters.append('run_at = %s')
             delta = (self.attempts**4) + 5
-            if self.max_backoff_delay_seconds > 0 and delta > self.max_backoff_delay_seconds:
+            if self.max_backoff_delay_seconds and delta > self.max_backoff_delay_seconds:
```

Comment:
Can we also set a minimum of 5 in case the user set the max to a very small number (e.g. 1).